### PR TITLE
Merge sugar and glucose, touch up how some drinks metabolize.

### PR DESCRIPTION
**Pull request by Elijahrane**
## About the PR 

**Changelog**

:cl: Rane
- tweak: Glucose and sugar are now the same thing.
- add: Alcohols now metabolize into some ethanol.
- tweak: Heavily caffeinated drinks metabolize into some theobromine instead of directly containing theobromine in their containers.

### CODE REVIEW

**Comment by mirrorcult** [comment]
idk if i understand the ethanol change

**Comment by Elijahrane** [comment]
> idk if i understand the ethanol change

Just googled the alcohol contents of all the drinks and divided in half since it's per 0.5u

**Comment by mirrorcult** [comment]
no i dont understand why its being done in the first place

**Comment by mirrorcult** [comment]
like what does it add

**Comment by Elijahrane** [comment]
All of the alcohol effects can be put on ethanol itself and avoid most of the jank of SS13 where to get drunk you need to just drink more types of drinks. Like, literally, if there was just 3% abv beer under 10 different names in ss13 then drinking 1/10th of the amount of each of them would be deadly but drinking all of one of them does nothing.

**Comment by mirrorcult** [comment]
okay, so basically just metabolizing into amounts of ethanol rather than defining the alcohol power on the reagent itself (i am currently working on adding alcohol effects and thats what i was doing)? i guess i'm okay with this

File: Resources/Prototypes/Reagents/Consumable/Food/food.yml [hunk 1]
```diff
@@ -15,23 +15,6 @@
...
-  color: "#ffffff"
-  boilingPoint: 340282300000000000000000000000000000000 #Fun fact: Glucose can't boil. So let's just set it to the maximum float value.
-  meltingPoint: 146.0
-  plantMetabolism:
-  - !type:PlantAdjustNutrition
-    amount: 0.1
-  - !type:PlantAdjustWeeds
-    amount: 2
-  - !type:PlantAdjustPests
-    amount: 2
```
Comment (juliangiebel) [inline Resources/Prototypes/Reagents/Consumable/Food/food.yml:32]: Shouldn't this be added to sugar if you remove Glucose? Is there a specific reason sugar shouldn't have those?

**Review by juliangiebel — CHANGES_REQUESTED** [review]
Looks good. There are just things missing from Sugar thet where set for Glucose

File: Resources/Prototypes/Reagents/Consumable/Food/food.yml (hunk 1 above)
Comment (Elijahrane) [reply Resources/Prototypes/Reagents/Consumable/Food/food.yml:32]: I added it back

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**